src/darwin/Framework/CHIP/templates/CHIPCommandPayloadsObjc.zapt

```diff
@@ -10,19 +10,25 @@ NS_ASSUME_NONNULL_BEGIN
 
 {{#zcl_clusters}}
 {{#zcl_commands}}
+@interface CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params : NSObject
 {{#zcl_command_arguments}}
-{{#first}}
-@interface CHIP{{asUpperCamelCase parent.parent.name}}Cluster{{asUpperCamelCase parent.name}}Params : NSObject
-{{/first}}
+
 {{! Override the getter name because some of our properties start with things
     like "new" or "init" }}
 @property (strong, nonatomic{{#unless (isStrEqual (asGetterName label) (asStructPropertyName label))}}, getter={{asGetterName label}}{{/unless}}) {{asObjectiveCType type parent.parent.name}} {{asStructPropertyName label}};
-{{#last}}
- - (instancetype)init;
-@end
-{{/last}}
 {{/zcl_command_arguments}}
+/**
+  * Whether a Timed Invoke Command Request is sent. Default is nil and is treated as
+  * if a cluster command is sent.
+  *
+  * If the property contains a value, the command will request a timed command within the
+  * specified time.
```

Comment:
```suggestion
 * Controls whether the command is a timed command (using Timed Invoke).
 * 
 * If nil (the default value), a regular invoke is done for commands that do
 * not require a timed invoke and a timed invoke with some default timed request
 * timeout is done for commands that require a timed invoke.
 *
 * If not nil, a timed invoke is done, with the provided value used as the timed
 * request timeout.  The value should be chosen small enough to provide the
 * desired security properties but large enough that it will allow a round-trip
 * from the sever to the client (for the status response and actual invoke
 * request) within the timeout window.
```